### PR TITLE
Add one-shot-deployment

### PR DESCRIPTION
**Pull request by dtitov**
Hi, everyone. After today's pain with deployment I decided to make our lives easier. For this purpose I created Docker image which does allllll the preparations and copies out required files (credits to @silverdaz for being patient and providing Makefile 😸 ). The only thing to specify manually is Central EGA password (see Slack for details). So the command to run is:

```
docker build -t lega-setup . && \
docker run -d -it --rm -v $(pwd):/configuration -e password=<Central EGA password> --name lega-setup lega-setup:latest && \
docker-compose up -d
```

Please, test it and feel free to make any suggestions. 

P.S. Linking it to my issue, because this is actually related to creating the CI pipeline.

### CODE REVIEW

**Comment by nanjiangshu** [comment]
well done @dtitov

**Comment by silverdaz** [comment]
I like the idea, and I have a suggestion for improvment.

Essentially, what you want is _stubbing_ (like you said) and not _lega-setup_, because LocalEGA setup _is_ `docker-compose up -d`. So, what we can do is prepare a separate docker-compose file, including all the stubbing parts and say ` docker-compose -f stubbing.yml up -d`, followed by `docker-compose up -d` (for the real setup of LocalEGA).

I'll prepare some code for it for illustration and add it to the conversation here.

**Comment by dtitov** [comment]
`lega-setup` is just a sample name for the container, probably not the best: it could be something like `lega-bootstrap`, to be more precise. It's like: "we are bootstrapping things first and then we are starting things" - that was my idea. 

But I'm okay with with `docker-compose -f stubbing.yml up -d` too, if it will be as simple as you described. The only thing to mention: I'm a bit dependent on this task, because in any case I need one of these approaches to continue with pipeline. So if you are going to make something like this, I kindly ask you not to postpone with this.

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+SSL_CERT=./ssl_home/ssl.cert \n\
+SSL_KEY=./ssl_home/ssl.key \n\
+RSA_SEC=./rsa_home/sec.pem \n\
+RSA_PUB=./rsa_home/pub.pem \n\
+GPG_HOME=./gpg_home \n\
+CEGA_USERS=./cega_users" >> .env
+
+RUN mkdir ../configuration
+
+CMD echo "password = ${password}" >> conf.ini && cp -r . ../configuration
```
Comment (viklund) [inline docker/Dockerfile]: Apart from this though, no newline?

**Comment by silverdaz** [comment]
Moreover: The GnuPG setup is [already there](https://github.com/NBISweden/LocalEGA/blob/master/docker/images/worker/Dockerfile)

**Pull request closed without merging**